addons/im_livechat/models/chatbot_script_question_answer.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+from odoo.osv import expression
+
+
+class ChatbotScriptQuestionAnswer(models.Model):
+    _name = 'im_livechat.chatbot.script_question_answer'
+    _description = 'Chatbot Script Question Answer'
+
+    name = fields.Char(string='Name', required=True)
+    sequence = fields.Integer(string='Sequence')
+    redirect_link = fields.Char('Redirect Link')
```

Comment:
Could be great to add an help on this one (usage ? local only or can we link external stuff ? ..)